src/drivers/net/emac/emac_desc.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file
+ *
+ * @date Nov 12, 2018
+ * @author Anton Bondarev
+ */
+
+#ifndef SRC_DRIVERS_NET_EMAC_EMAC_DESC_H_
+#define SRC_DRIVERS_NET_EMAC_EMAC_DESC_H_
+
+
+
+/**
+ * EMAC Buffer Descriptor
+ */
+struct emac_desc {
+	uint32_t next;
+	uint32_t data;
+#if __BYTE_ORDER == __BIG_ENDIAN
```

Comment:
I think it's better to unite this `#if .. _BIG_ENDIAN` with the following one, i.e.

```
#if __BYTE_ORDER == __BIG_ENDIAN
	uint16_t data_off;
	uint16_t data_len;
	uint16_t flags;
	uint16_t packet_len;
#elif __BYTE_ORDER == __LITTLE_ENDIAN
	uint16_t data_len;
	uint16_t data_off;
	uint16_t packet_len;
	uint16_t flags;
#endif
```